docs/fxa-engineering/fxa-sub-platform.md

```diff
@@ -169,13 +169,120 @@ server:
+Some Stripe webhooks will trigger emails.  These emails are behind a feature flag.  If you wish to send emails in your environment, set the auth server configuration
+
+```
+{
+  "subscriptions": {
+    "transactionalEmails": {
+      "enabled": true
+    }
+  }
+}
+```
+
+or the environment variable `SUBSCRIPTIONS_TRANSACTIONAL_EMAILS_ENABLED` to "true".  In order to receive Stripe webhook events in your local development, you need to use the [Stripe CLI](https://stripe.com/docs/stripe-cli/webhooks)'s event forwarding feature.
+
+## PayPal Integration
+
+PayPal can be configured as an additional payment provider in the Subscription Platform.  PayPal's [Express Checkout Reference Transactions](https://developer.paypal.com/docs/archive/express-checkout/integration-guide/ECReferenceTxns/) is the feature that enables the Subscription Platform to use PayPal as a payment provider for recurring subscriptions.  The customer's PayPal Billing Agreement ID is saved for future subscription invoices.  See [the diagram below](#paypal-checkout) for details on this process.
+
+The PayPal paid subscriptions are still driven by Stripe's subscription and invoicing model.  The key difference is that PayPal paid subscriptions have a [collection method](https://stripe.com/docs/api/subscriptions/object#subscription_object-collection_method) of `send_invoice`.  The [PayPal processor](#paypal-processor) is used to pay the invoices with the customer's billing agreement ID.
+
+### PayPal Accounts
+
+You need three types of PayPal accounts for development.
+
+- PayPal Developer Account: allows you to access the PayPal Developer Dashboard
+- Sandbox PayPal Personal Account: used for testing as the customer
+- Sandbox PayPal Business Account: used for testing
+
+To create a PayPal developer account, sign up at https://developer.paypal.com/.  Note that if you are a Mozilla employee, you should contact the Mozilla PayPal admin in Finance to set up a developer account.  Additionally, you should [enable 2FA for the developer account](https://www.paypal.com/businessmanage/profile/loginSecurity).
+
+Once you are in the PayPal developer dashboard, navigate to "Accounts" under the Sandbox section of the menu.  Here you can create [a pair of personal and business sandbox accounts](https://developer.paypal.com/docs/api-basics/sandbox/accounts/).  (To easily create multiple accounts for testing, there's a [bulk account creation feature](https://developer.paypal.com/docs/api-basics/sandbox/bulk-accounts/).)
+
+Once you've added your account pair, navigate to the bussines account by selecting "View/edit account".  Now click on the "API Credentials" tab.  You'll need the "NVP/SOAP Sandbox API Credentials" for the next section.
+
+### Configuration
+
+#### Auth Server
+
+In order to enable and use PayPal in the auth server, set the following configuration options
+
+```
+{
+  "subscriptions": {
+    "paypalNvpSigCredentials": true,
```

Comment:
Paypalnvpsigcredentials is actually an object with the other 4 options under it.

Reply:
Oops.  Good catch.  I copied from the convict declaration, and obviously failed at editing it. 🤦